### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  com.microsoft.azure.toolkit.lib.auth.exception.AzureToolkitAuthenticationException: you are not signed-in.

### ISSUE THREAD

**Issue by vitaliy-serdyukov** [microsoft/azure-tools-for-java#6725]
IntelliJ build version: 2022.1.2 IU-221.5787.30
OS: Windows 10
JDK: JetBrains s.r.o. 11.0.15
Plugin version: 3.65.2-2022.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,7,7,1281x681,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=com.microsoft.azure.toolkit.lib.auth.exception.AzureToolkitAuthenticationException: you are not signed-in.
	at com.microsoft.azure.toolkit.lib.auth.AzureAccount.lambda$account$0(AzureAccount.java:61)
	at com.microsoft.azure.toolkit.lib.auth.AzureAccount.account(AzureAccount.java:61)
	at com.microsoft.azure.toolkit.lib.auth.AzureAccount.account(AzureAccount.java:51)
	at com.microsoft.azure.toolkit.lib.common.model.AbstractAzResourceModule.get(AbstractAzResourceModule.java:191)
	at com.microsoft.azure.toolkit.lib.resource.AzureResources.groups(AzureResources.java:28)
	at com.microsoft.azure.toolkit.lib.resource.ResourceGroupConfig.toResource(ResourceGroupConfig.java:43)
	at com.microsoft.azure.toolkit.intellij.legacy.webapp.runner.webappconfig.slimui.WebAppSlimSettingPanel.lambda$apply$2(WebAppSlimSettingPanel.java:150)
	at com.microsoft.azur

**Comment by wangmingliang-ms** [comment]
thanks for reporting, this bug will be fixed in the coming v3.66.0

**Comment by wangmingliang-ms** [comment]
fixed in v3.66.0
